content/en/integrations/lacework.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Lacework
+name: lacework
+kind: integration
+description: "Forward your Lacework logs to Datadog."
+short_description: "Forward your Lacework logs to Datadog."
+categories:
+- log collection
+- security
+doc_link: /integrations/lacework/
+has_logo: true
+integration_title: lacework
+is_public: true
+public_title: Datadog-Lacework Integration
+---
```

Comment:
Check https://docs-staging.datadoghq.com/gus/lacework/integrations/lacework/

The title is showing with a lowercase `l`. Not sure which one changes this.